app/bundles/EmailBundle/Model/EmailModel.php

```diff
@@ -474,17 +466,26 @@ protected function dispatchEvent($action, &$entity, $isNew = false, Event $event
     }
 
     /**
-     * @param bool $viaBrowser
+     * @param Stat|string|null $stat                    The null is just for BC reasons, should be Stat|string
+     * @param bool             $throwDoctrineExceptions in asynchronous processing; we do not wish to ignore the error, rather let the messenger do the handling
      *
-     * @throws \Exception
-     */
-    public function hitEmail($stat, $request, $viaBrowser = false, $activeRequest = true)
-    {
+     * @throws OptimisticLockException|\Exception
+     */
+    public function hitEmail(
+        $stat,
+        ?Request $request,
```

Comment:
```suggestion
        Request $request,
```
As the body of the method does not contain any check whether the `$request` is (not) `null`, we should not allow passing null for that parameter.